protocols/grpc/agent.proto

```diff
@@ -194,7 +194,7 @@ message OnlineCPUMemRequest {
 }
 
 message Storage {
-	string driver = 1;           // empty in most cases. it will support "drbd", "bcache" ...
+	string driver = 1;           // Empty driver field means that we expect to do a mount. If driver field is not empty, we expect to do an operation related to a device. The type of device should be given through this field, triggering a specific function from the agent code.
```

Comment:
please remove "Empty driver field means that we expect to do a mount. "
the driver field is needed when the agent's general mount code can't handle it,
for example special devices, fstypes...

Reply:
PR updated !